src/features/presence/PresenceLink.js

```diff
@@ -39,11 +39,7 @@ class PresenceLink extends Component {
                     <PlainButton>{children}</PlainButton>
                 </div>
                 <Overlay shouldDefaultFocus={false}>
-                    <PresenceDropdown
-                        className="presence-dropdown"
```

Comment:
Is this referenced in the consuming application(s) at all?

Reply:
Not that I've seen